playbulb-candle/sw.js

```diff
@@ -23,6 +27,6 @@ self.addEventListener('activate', function(event) {
         cacheNames.filter(cacheName => (cacheName !== 'playbulb-candle'))
                   .map(cacheName => caches.delete(cacheName))
       );
-    });
+    })
```

Comment:
Keeping the semicolon raises `Uncaught SyntaxError: missing ) after argument list`